app/views/admin/reports/items_in_maintenance/index.html.erb

```diff
@@ -34,6 +34,10 @@
             <%= f.label :status_eq, "Ticket Status" %>
             <%= f.select :status_eq, [["Assess", "assess"], ["Parts", "parts"], ["Repairing", "repairing"]], {include_blank: true} %>
           </div>
+          <div class="filter-field tag">
+            <%= f.label :tag_eq, "Tag" %>
+            <%= f.select :tag_eq, ActsAsTaggableOn::Tag.most_used.pluck(:name), {include_blank: true, selected: params[:q][:tag_eq]} %>
```

Comment:
`selected: params[:q][:tag_eq]` -- this is required for some reason. When it's removed, the page fails and returns this error: 

> NoMethodError: undefined method 'tag_eq' for an instance of Ransack::Search